crates/core/src/test.rs

```diff
@@ -12833,7 +12833,7 @@ fn rust_match_fn_params() {
                 |impl Matcher for FilePattern {
                 |    fn execute<'a>(
                 |        &'a self,
-                |        resolved_pattern: &ResolvedPattern<'a>,
+                |        resolved_pattern: &Q::ResolvedPattern<'a>,
```

Comment:
big find and replace casualty?  Change is fine, just curious why.

Reply:
Ah, probably:)